### DISCUSSION TITLE
New types of adds on youtube

### DISCUSSION THREAD

**Discussion by AzRenoxM** [ghostery/adblocker#4145] [Ideas]
Hi guys, 
yesterday I wanted to watch something on youtube, and suddenly instead of my selected video, appeared an add, it was formatted like a typical video, add blocker obviously haven't worked, since it was interpreting this add as a normal video, but I have an idea on how to make the blocker differentiate between an add and a video itself, when watching some video you can skip it 5 seconds ahead, and on an add you never will, I'm not good in the browser technologies but maybe this idea could be useful, so like "block the video that can't be skipped".
Good luck

**Comment by seia-soto** [comment]
Hi there,

Sorry for the late reply. We're not actively monitoring this place. We're also sorry to hear that you had an inconvenience with our product. In the future, you can visit https://github.com/ghostery/broken-page-reports to report broken pages by creating new issue.

Best
